samples/Perl 6/Any.pod6

```diff
@@ -0,0 +1,987 @@
+=begin pod
```

Comment:
This file needs to be removed too as you've removed `.pod6` from this PR.

Reply:
Done.